src/test/regress/bin/normalize.sed

```diff
@@ -200,6 +200,10 @@ s/relation with OID [0-9]+ does not exist/relation with OID XXXX does not exist/
 # normalize storage id of columnar tables
 s/^storage id: [0-9]+$/storage id: xxxxx/g
 
+# normalize PREPARE TRANSACTION and COMMIT PREPARED
```

Comment:
PREPARE TRANSACTION / COMMIT PREPARED are not intended to be printed. If we add this rule, we'll allow any further tests to print these commands, which doesn't sound like a good idea to me.

 I think we should add the regression tests similar to this file: https://github.com/citusdata/citus/blob/21fd2e2c92bc462aded1ad5f39dbdeb25ea198f0/src/test/regress/sql/multi_transaction_recovery.sql

Where we disable 2PC recovery, then execute the commands and check `pg_dist_transaction`.


Note that, if we do that, we cannot run the the test file in parallel with any other test as they might become flaky because of disabling 2PC recovery. 

We could even move these tests there.